packages/block-library/src/navigation/edit/responsive-wrapper.js

```diff
@@ -10,11 +10,14 @@ import { close, Icon } from '@wordpress/icons';
 import { Button } from '@wordpress/components';
 import { __ } from '@wordpress/i18n';
 import { getColorClassName } from '@wordpress/block-editor';
+import { useSelect } from '@wordpress/data';
+import { store as coreStore } from '@wordpress/core-data';
 
 /**
  * Internal dependencies
  */
 import OverlayMenuIcon from './overlay-menu-icon';
+import { createTemplatePartId } from '../../template-part/edit/utils/create-template-part-id';
```

Comment:
Given that we are depending on external utils, we need to add some test coverage here.

I would add a quick unit tests that validates the behaviour the Nav block expects for `createTemplatePartId`. That way if the original implementation changes _at all_ then these tests will catch it. We'll need to name the tests really well so that if they were to fail the PR author would understand that they can't simply modify the test because it is documenting behaviour expected by the Nav block.